keyboards/macroflow_original/keymaps/default/rules.mk

```
@@ -0,0 +1,3 @@
+VIA_ENABLE = yes
+ENCODER_MAP_ENABLE = yes
+OLED_ENABLE = yes
```

VIA cannot be enabled on default keymaps, and the workflow no w is to submit an additional keymap to <https://github.com/the-via/qmk_userspace_via>.

```suggestion
ENCODER_MAP_ENABLE = yes
```